src/context/AuthenticatedHydrators.tsx

```diff
@@ -1,8 +1,15 @@
 import { EntitiesHydrator } from 'stores/Entities/Hydrator';
 import { BaseComponentProps } from 'types';
+import { TenantBillingDetailsContextProvider } from './fetcher/Tenant';
 
 function AuthenticatedHydrators({ children }: BaseComponentProps) {
-    return <EntitiesHydrator>{children}</EntitiesHydrator>;
+    return (
+        <EntitiesHydrator>
+            <TenantBillingDetailsContextProvider>
```

Comment:
Now we have something to fetch just the payment details of tenants the user has admin access to.